client/helpers/user-helpers.js

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
I remember this comment time ago, about custom publication user.js, ok, I remove this unnecessary comment